packages/workflow/package.json

```diff
@@ -84,24 +84,24 @@
     "pretty-ms": "^5.0.0",
     "raf": "^3.4.0",
     "react-app-polyfill": "^1.0.1",
-    "react-hot-loader": "^4.3.6",
+    "react-hot-loader": "^4.12.17",
     "react-hot-loader-loader": "0.0.7",
     "regenerator-runtime": "^0.13.1",
     "require-relative": "^0.8.7",
-    "sass-loader": "^7.0.1",
+    "sass-loader": "^8.0.0",
```

Comment:
This is a little scary. I had done something previously and it worked locally but when going through our CI process it failed.

Check the breaking changes and make sure its not going to interfere with our current configs:
https://github.com/webpack-contrib/sass-loader/blob/master/CHANGELOG.md#800-2019-08-29